internal/server/server.go

```diff
@@ -302,7 +302,6 @@ func registerUIRoutes(router *gin.Engine, opts UIOptions) {
 		req.URL.Scheme = remote.Scheme
 		req.URL.Host = remote.Host
 	}
-	proxy.ErrorLog = log.New(logging.NewFilteredHTTPLogger(), "", 0)
```

Comment:
I forget if we use this proxy in production or not, but maybe we want the `ErrorLog` here as well?

Reply:
Great catch!